src/liballoc/vec.rs

```diff
@@ -2903,7 +2901,6 @@ impl<T> FusedIterator for Drain<'_, T> {}
 /// documentation for more.
 ///
 /// [`splice()`]: struct.Vec.html#method.splice
```

Comment:
```suggestion
/// [`splice()`]: Vec::splice
```

Reply:
Shouldn't we use?
```rust
/// This struct is created by the [`Vec::splice`]. See its
```

Reply:
Sure, that works too. I was trying to avoid changing the text but if you think that looks nicer we can go with that.

I would say `// This struct is created by [`Vec::splice()`]. See its` though.